KlaviyoSDKDependencies.podspec

```diff
@@ -0,0 +1,15 @@
+Pod::Spec.new do |s|
+  s.name             = "KlaviyoSDKDependencies"
+  s.version          = "4.0.0"
+  s.summary          = "Dependency for the Klaviyo SDK"
+  s.description      = <<-DESC
+                        Klaviyo external dependencies all rolled in one package
+                       DESC
+  s.homepage         = "https://github.com/klaviyo/klaviyo-swift-sdk"
+  s.license          = { :type => "MIT", :file => "LICENSE" }
+  s.author           = { "Mobile @ Klaviyo" => "mobile@klaviyo.com" }
+  s.source           = { :git => "https://github.com/klaviyo/klaviyo-swift-sdk.git", :tag => s.version.to_s }
```

Comment:
I'm probably not looking in the right file, but I am curious what dependencies are going in this. I saw in a lower file that both core and this were imported, why is that? Is this where the dependencies of our SDK live?

Reply:
We shouldn't have any dependencies after this change in the sdk properly. I've internned pretty much everything except in tests which uses combine schedulers and a few other misc things.